src/compile/config.ts

```diff
@@ -83,32 +83,23 @@ export function orient(mark: Mark, encoding: Encoding, markConfig: MarkConfig =
       }
       // y:Q or Ambiguous case, return horizontal
       return Orient.HORIZONTAL;
+
     case RULE:
-      if (xIsRange) {
-        return Orient.HORIZONTAL;
-      }
-      if (yIsRange) {
```

Comment:
Here you use if + return, below you use else if. I prefer to use elseif to indicate exclusiveness.